libs/input-mapping/src/Helper/LoadTypeDecider.php

```diff
@@ -21,20 +23,35 @@ public static function canClone(array $tableInfo, string $workspaceType, array $
         return true;
     }
 
-    public static function canUseView(array $tableInfo, string $workspaceType, array $exportOptions): bool
-    {
-        if ($tableInfo['isAlias']) {
-            return false;
-        }
-
+    public static function canUseView(
+        array $tableInfo,
+        string $workspaceType,
+        array $exportOptions,
+        string $currentProjectId,
+    ): bool {
         if (($tableInfo['bucket']['backend'] !== $workspaceType) ||
             ($workspaceType !== 'bigquery')
         ) {
             return false;
         }
 
         if ($exportOptions && array_keys($exportOptions) !== ['overwrite']) {
-            return false;
+            throw new InvalidInputException(sprintf(
```

Comment:
Mi tu smrdi ze to je decider a validator v jednom :(

Reply:
Úplně souhlasím, mě se to taky nelíbí, upravím to